content/open-source/wolfi/wolfi-with-dockerfiles.md

```diff
@@ -110,10 +97,12 @@ Copy the following content to it:
 ```
 FROM cgr.dev/chainguard/wolfi-base
```

Comment:
I'm fine with this, I just wonder if using python:$version-dev would be simpler?